package.json

```diff
@@ -4,15 +4,15 @@
     "description": "Backend for Find a Doc, Japan",
     "main": "index.js",
     "scripts": {
-        "dev": "echo '🧶 Installing npm dependencies...🧶' && npm install && npm run generate && npm run dev:startserver",
-        "dev:startserver": "echo '💻 💻 💻✨ Starting dev server...' && cross-env NODE_ENV=development nodemon -e 'ts' --exec 'node --inspect --loader ts-node/esm src/index.ts'",
+        "dev": "echo '🧶 Installing yarn dependencies...🧶' && yarn && yarn generate && yarn dev:startserver",
+        "dev:startserver": "echo '💻 💻 💻✨ Starting dev server...' && cross-env NODE_ENV=production nodemon -e 'ts' --exec 'node --inspect --loader ts-node/esm src/index.ts'",
```

Comment:
Why are we pointing the dev server to `NODE_ENV=production`? 🤔

Reply:
oh good catch! updated